lib/workers/repository/config-migration/pr/errors-warnings.ts

```diff
@@ -0,0 +1,28 @@
+import type { RenovateConfig } from '../../../../config/types';
+
+export function getWarnings(config: RenovateConfig): string {
+  if (!config?.warnings?.length) {
```

Comment:
```suggestion
  if (!config.warnings?.length) {
```

You shoul never pass `null` or `undefined` for config